internal/load/load.go

```diff
@@ -32,13 +32,13 @@ func Checks(ctx context.Context, enabledChecks []string, experimentalChecks []st
 	if isEnabled(enabledChecks, "owners") {
 		var cfg struct {
 			OwnerChecker check.ValidOwnerConfig
-			Github       github.ClientConfig
+			GitHub       github.ClientConfig
 		}
 		if err := envconfig.Init(&cfg); err != nil {
 			return nil, errors.Wrapf(err, "while loading config for %s", "owners")
 		}
 
-		ghClient, err := github.NewClient(ctx, cfg.Github)
+		ghClient, err := github.NewClient(ctx, cfg.GitHub)
```

Comment:
I think this is a harmless localized change

Reply:
Yes, you are right. It will change `GITHUB_ACCESS_TOKEN` to `GIT_HUB_ACCESS_TOKEN` so I just reverted this change 👍  besides that, it LGTM 👌